frontend/packages/console-app/src/components/cloud-shell/__tests__/CloudShellDrawer.spec.tsx

```diff
@@ -18,6 +18,21 @@ describe('CloudShellDrawerComponent', () => {
     ).toEqual('<p>Terminal content</p>');
   });
 
+  it('should have techPreview badge in header', () => {
+    const wrapper = shallow(
+      <CloudShellDrawer onClose={() => null}>
+        <p>Terminal content</p>
+      </CloudShellDrawer>,
+    );
+    expect(
+      wrapper
+        .find(Drawer)
+        .shallow()
+        .find(InlineTechPreviewBadge)
+        .exists(),
+    ).toBe(true);
+  });
```

Comment:
Not sure this is a good test. You're testing for static content. I think this is akin to testing we have the title "OpenShift command line terminal", which won't really help us ensure the sanity of the code.

Reply:
will remove these